pkg/generator/templates/cloud-init.gardenlinux.template

```diff
@@ -6,6 +6,16 @@ cat << EOF | base64 -d > '{{ .Path }}'
 EOF
 {{- end -}}
 
+{{ if isContainerDEnabled .CRI -}}
```

Comment:
```suggestion
{{ if and (isContainerDEnabled .CRI)  (not .Bootstrap) -}}
```

Reply:
Otherwise file is included in cloud-config-downloader and the cloud-config-execution script.

Reply:
done